mac

```diff
@@ -128,9 +128,11 @@ if ! brew_is_installed "node"; then
     fancy_echo "See https://pages.18f.gov/frontend/#install-npm"
   elif ! command -v nvm > /dev/null; then
     fancy_echo 'Installing nvm and lts Node.js and npm...'
-    touch ~/.bash_profile
-    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.7/install.sh | bash
-    nvm install --lts
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.32.1/install.sh | bash
+    export NVM_DIR="$HOME/.nvm"
+    # shellcheck source=/dev/null
+    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Comment:
I was running this on a local branch and still seeing issues when running `nvm install node --lts` on the previous PR #81. If I ran lines 134 and 135 manually after this script ran, it would install properly. It may be related to the first line in this script `#!/bin/sh`. It seems like we need this script to run as the user who is installing `nvm` rather than just using `sh` to execute the script.

Some light research below.
creationix/nvm#1290 (**npm is only available in interactive shell**)
http://unix.stackexchange.com/questions/321176/command-not-found-when-running-script-as-other-user